app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+YMIN = 1130000
+# this would be the max extent for CH
+# YMAX = 1252000
+YMAX = 1140000
+
+MIN_DATETIME = isoparse('1970-01-01T00:00:01Z')
+MAX_DATETIME = isoparse('2020-12-31T23:59:10Z')
+
+
+def random_datetime(start, end):
+    """Generate a random datetime between `start` and `end`"""
+    return start + datetime.timedelta(
+        # Get a random amount of seconds between `start` and `end`
+        seconds=random.randint(0, int((end - start).total_seconds())),
+    )
+
+
+class Command(BaseCommand):
+    help = """Populates the local test database with dummy data
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--populate',
+            action='store_true',
+            help='Populate dummy data',
+        )
+
+        parser.add_argument(
+            '--clean',
+            action='store_true',
+            help='Clean dummy data',
+        )
+
+    def handle(self, *args, **options):
+
+        geo = Polygon.from_bbox((2570000.0, 1129970.0, 2580000.0, 1139990.0))
+        geo.srid = 2056
+        geo.transform(4326, clone=False)
+        range(2570000, 2746000, 1000)
+        range(1130000, 1252000, 1000)
+
+        if options['clean']:
+            Collection.objects.filter(name__startswith='perftest').delete()
+        elif options['populate']:
+
+            for collection_id in [f'perftest-collection-{x}' for x in range(30)]:
+                collection, _ = Collection.objects.get_or_create(
+                    name=collection_id,
+                    defaults={
+                        'description': 'This is a description',
+                        'license': 'test',
+                        'summaries': {
+                            "eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []
+                        },
+                        'title': 'Test title'
+                    }
+                )
+                for _xmin in range(XMIN, XMAX, 1000):
+                    for _ymin in range(YMIN, YMAX, 1000):
+                        geo = Polygon.from_bbox((_xmin, _ymin, _xmin + 1000, _ymin + 1000))
+                        geo.srid = 2056
+                        geo.transform(4326, clone=False)
+                        item_id = f'perftest-item-{_xmin}-{_ymin}'
+
+                        item, _ = Item.objects.get_or_create(
+                            collection=collection,
+                            name=item_id,
+                            defaults={
+                                'properties_datetime': random_datetime(MIN_DATETIME, MAX_DATETIME),
+                                'properties_title': "My Title",
+                                'geometry': geo
+                            }
+                        )
+
+                        for asset_id in [f'perftest-asset-{x}' for x in range(3)]:
+                            asset, _ = Asset.objects.get_or_create(
+                                item=item,
+                                name=asset_id,
```

Comment:
Asset ID should have the media type extension, otherwise this won't work anymore when https://jira.swisstopo.ch/browse/BGDIINF_SB-1566 will be implemented